packages/sanity/src/structure/documentActions/PublishAction.tsx

```diff
@@ -41,6 +41,9 @@ function getDisabledReason(
 function AlreadyPublished({publishedAt}: {publishedAt: string}) {
   const {t} = useTranslation(structureLocaleNamespace)
   const timeSincePublished = useRelativeTime(publishedAt)
+  if (timeSincePublished === 'yesterday') {
+    return <span>{t('action.publish.already-published-yesterday.tooltip')}</span>
+  }
```

Comment:
This only catches the case when english is used. For other languages it will not match and have the same issue. Eg in Norwegian, the `useRelativeTime` hook would return `i går`, and so the if-branch would not be hit and it would result in `Publisert i går siden`.

I think this is better solved by passing the `useTemporalPhase` option to the `useRelativeTime` hook and removing "ago" from the resource string. That should automatically include "ago" if necessary, and do the right thing for other languages - eg `Publisert i går` in norwegian, and `Publisert for 3 døgn siden` for dates further back.

```suggestion
  const timeSincePublished = useRelativeTime(publishedAt, {useTemporalPhrase: true})
```

Reply:
Note: unfortunately this also means we have to update all the locales to not also include the temporal phrase in their translations, but at least this will be consistent with the other places within the studio that we do this.

Reply:
Ahh, so `useRelativeTime` deals with locales. I pushed your suggestion and tested with different locales and it looks accurate! 

About your note - this will be updated when I update the `resources.ts`, correct? All other locales use this as it's source?

Reply:
> About your note - this will be updated when I update the `resources.ts`, correct? All other locales use this as it's source?

Yep!